.circleci/config.yml

```diff
@@ -102,6 +102,9 @@ env-mac-large: &env-mac-large
 env-disable-crash-reporter-tests: &env-disable-crash-reporter-tests
   DISABLE_CRASH_REPORTER_TESTS: true
 
+env-ninja-status: &env-ninja-status
+  NINJA_STATUS: "[%r processes, %f/%t @ %o/s : %es]"
```

Comment:
hm, i see now that this stuff comes from autoninja, which also has that extra space there? I wonder if we should just invoke `autoninja`. either way i was probably wrong about removing the space.
```suggestion
  NINJA_STATUS: "[%r processes, %f/%t @ %o/s : %es ]"
```

Reply:
It works either way 😁